openlibrary/templates/interstitial.html

```diff
@@ -1,16 +1,20 @@
-$def with(url, provider="", wait=5)
+$def with(url, provider_name="", wait=5)
 
-<h1>$_("You are being redirected to your book")</h1>
+$if provider_name is None:
+  $ provider_name = ""
 
-<p id="countdown">$_("This book is provided by %s, a third-party Open Library Trusted Book Provider", "<strong>" + provider= "</strong>")</p>
+<div class="interstitial">
```

Comment:
```suggestion
<div id="contentBody" class="interstitial">
```
The styling that is applied to `#contentBody` should be enough:

![image](https://github.com/user-attachments/assets/fe58c322-b50d-4df9-b7ad-05879ec4fff9)